### ISSUE TITLE
Create CodeMeta ontology

### ISSUE THREAD

**Issue by gothub** [codemeta/codemeta#88]
This OWL2 ontology will contain the CodeMeta concepts and is intended to be used to 
verify CodeMeta instance files. First the instance files have to be serialized to RDF using
the JSON-LD API and then verified against the ontology. Serializing instance files to RDF is
not required for "regular usage" of CodeMeta, so this serialization will initially be included
in the github test suite.

**Comment by gothub** [comment]
I've reconsidered using RDF serialized CodeMeta instance files for verification. Reasons it isn't a good solution:

- we don't have any control of the json-ld to RDF serialization
- the JSON terms are converted to their URI equivalent in the serialization
   Users won't know these names unless they look in the CodeMeta context file
- error messages from OWL2 validation will be in RDF terms, quite different from the source JSON file

[JSON Schema](http://json-schema.org/) provides a better alternative to verifying instance files. Benefits of JSON Schema:

- JSON Schema is like XSD for JSON, it verifies the source instance file structure and content
- the JSON Schema validator library phrases errors in terms of the source document, so it's easier for the user to remedy problems

JSON Schema has not been adopted as a standard, but it is widely used and is an active project.

I've checked in  a [CodeMeta JSON Schema file](https://github.com/codemeta/codemeta/blob/master/codemeta-json-schema.json) into the repo and am validating example documents with it.
